jepsen/src/jepsen/nemesis/time.clj

```diff
@@ -72,6 +80,9 @@
   (reify nemesis/Nemesis
     (setup! [nem test]
       (c/with-test-nodes test (install!))
```

Comment:
You can make this faster by folding both (install!) and stopping the NTP service inside a single with-test-nodes.